<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/Apix.png</normaloff>images/Apix.png</iconset>
  </property>
  <widget class="QWidget" name="mainLayout">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>106</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="selectSourceLabel">
       <property name="text">
        <string>Sélectionnez la source de données (*.csv):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="filePathWidget">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string extracomment="Parcourir le système de fichiers"/>
         </property>
         <property name="text">
          <string>Parcourir</string>
         </property>
         <property name="icon">
          <iconset theme="folder">
           <normaloff>../../../.designer/backup</normaloff>../../../.designer/backup</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="scoreLabel">
         <property name="text">
          <string>Niveau seuil (%) :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="scoreSpinBox">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>70</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="certifiableRadio">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Certifiables</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="nonCertifiableRadio">
         <property name="text">
          <string>Non certifiables</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QListView" name="pupilList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>256</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="pupilDetail">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>511</width>
      <height>411</height>
     </rect>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
